p2p/module.go

```diff
@@ -373,21 +382,42 @@ func (m *p2pModule) readStream(stream libp2pNetwork.Stream) {
 		m.logger.Debug().Err(err).Msg("setting stream read deadline")
 	}
 
+	// debug logging: stream scope stats
+	// (see: https://pkg.go.dev/github.com/libp2p/go-libp2p@v0.27.0/core/network#StreamScope)
+	// TECHDEBT: `logger.Global` is not a `*module.Logger`
```

Comment:
Can we not go with the same approach you have elsewhere? `logScope := LogScopeStatFactory(&logger.Global.Logger, "host transient resource scope")`